src/components/TaskCard/EditTaskCard.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+import { sanitizeText } from "utils/helpers/sanitizeText.js";
+import { editTask, completeTask } from "store/actions/";
+import style from "components/TaskCard/index.module.scss";
+import { ICON_DELETE, DELETE_ICON_ALT_TEXT, KEY_ENTER } from "utils/constant";
+import { ICON_COMPLETE, COMPLETE_ICON_ALT_TEXT } from "utils/constant/images";
+
+const EditTaskCard = ({ id, task, editableTasks, onEditableTasks }) => {
+  const [inputText, setInputText] = useState(task);
+  const [error, setError] = useState("");
+  const dispatch = useDispatch();
+
+  function removeFromEditList() {
+    const updatedList = editableTasks.filter((taskId) => taskId !== id);
+    onEditableTasks(updatedList);
+  }
+
+  function handleInputText(event) {
+    setInputText(event.target.value);
+  }
+
+  function handleSaveClick() {
```

Comment:
```suggestion
  function onSave() {
```

Reply:
bhai can we rename it handleSaveTask ?

![Screenshot 2023-05-09 at 10 02 29 AM](https://user-images.githubusercontent.com/126457859/236991115-b8b699d9-ea71-4e80-a4d0-340a17e10868.png)